ehitchhiking-ui/src/app/shared/api-services/black-list-api.service.ts

```diff
@@ -0,0 +1,75 @@
+import {Injectable} from '@angular/core';
+import {User} from '@shared/models/user';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+import {ApiService} from '@shared/services/api.service';
+import {Car} from '@shared/models/car';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class BlackListApiService {
+	BLACKLIST_PASSENGERS: User[] = [];
```

Comment:
there is no need to create local variables here. Do it in the methods

Reply:
done